docs/Comparison.md

```diff
@@ -2,23 +2,24 @@
-|Livepatching service                        | KernelCare               | Oracle Ksplice, KernelCare Available           | KernelCare Available                        | Not available     | KernelCare, Kpatch |
-|FIPS compliance                             | Yes                      | Yes                                            | Planned                                     | Not available     | Not available     |
-|ARM support                                 | Yes                      | Yes                                            | Yes                                         | Yes               | Yes               |
-|IBM PowerPC support                         | Yes                      | No                                             | Yes (versions ≥ 9)                          | Yes               | Yes               |
-|IBM Z (s390x) support                       | Yes                      | No                                             | Yes (versions ≥ 9)                          | No                | No                |
-|SecureBoot                                  | Yes                      | Yes                                            | Yes                                         | Yes               | Yes               |
-|Owned By:                                   | AlmaLinux OS Foundation  | Oracle Inc                                     | Rocky Enterprise Software Foundation        | Red Hat Inc       | Red Hat Inc       |
-|Owned by org type:                          | Non-Profit 501(c)6       | For Profit C-Corp                              | For Profit, Public Benefit Corp             | For Profit C-Corp | For Profit C-Corp |
+> Chart by [Carl George](https://www.linkedin.com/in/carlwgeorge/)
+
+![relationship_chart](/images/Comparison-relationship_chart.png)
+
+The chart above illustrates the relationship between Fedora, CentOS Stream, and RHEL.<br>
+AlmaLinux utilizes package sources from both RHEL and CentOS Stream to build its distribution. In contrast, Oracle Linux and Rocky Linux rely exclusively on package sources from RHEL.
+
+|Benchmarking against RHEL       |AlmaLinux                                 |Oracle Linux                        |Rocky Linux                                                           |CentOS Stream                  |RHEL                           |
+|--------------------------------|------------------------------------------|------------------------------------|----------------------------------------------------------------------|-------------------------------|-------------------------------|
+|First release                   |Since March 2021                          |Since 2006                          |Since June 2021                                                       |Since 2019                     |March 2002                     |
```

Comment:
Since we're renaming this row to "first release", we can probably drop the word "since" in each column.

On the dates, we can round things out by including the months for CentOS Stream and Oracle.

- CentOS Stream was first released in [September 2019](https://lists.centos.org/hyperkitty/list/announce@lists.centos.org/thread/AVVPYZLLYUE3GIDP3DIKTZ55X5T3YM4Y/)
- Oracle Linux was first released in [October 2006](https://web.archive.org/web/20070714050930/http://www.oracle.com/corporate/press/2006_oct/Oracle-Linux-Program.html)

Reply:
Thank you very much! I added the changes.